index.html

```diff
@@ -90,35 +92,18 @@ <h3 data-fx-fade><time datetime="2015-09-04T18:00:00Z">Friday, September 4th<br>
 		</header>
 
 		<main role="main" class="main">
-			<section class="tickets background-secondary" id="tickets">
-				<div class="tickets__inner">
-					<h2>Tickets</h2>
-
-					<div class="ticket">
-						<p class="ticket__description"><strong>We are sold out.</strong><br/><span class="ticket__description--extended">Thanks for being part of our journey this year again!</span></p>
-
-						<p class="ticket-form">
-							<a class="button button--big button--tickets is-inverted" href="https://ti.to/nightlybuild/nightlybuild-2015">
-								<span class="button__label"><s>Ticket Price 49€</s></span>
-							</a>
-						</p>
-					</div>
-
-					<hr>
-
-					<h3 id="diversity-tickets">NightlyBuild 2015 Diversity Tickets</h3>
-
-					<p>Our diversity ticket application is closed now. We will get back to the applicants in the next few days.<br/><strong>Thanks for all your support!</strong></p>
-
-					<p>We’re happy to be supported by an independent jury: <strong>Hanne Hofmann, Jeanette and Andreas Klein</strong></p>
-				</div>
-			</section>
-
 			<section class="theme" id="theme" data-fx-fade>
 				<div class="theme__inner">
 					<h2>Central Theme</h2>
 					<p>This year’s central theme is ‘Work efficient and live your life’. With our speaker and topic selection we want to give you a guide on how to improve the holy grail of ‘work-life balance’.</p>
 					<p>You will learn how to work more efficient with sane approaches to front-end tooling, hear how to find your niche in our industry, listen to building side projects while caring about your children and husband / wife and get advice how to stay up to date in todays’ fast-moving world.</p>
+
+					<div class="embed">
+						<iframe src="https://player.vimeo.com/video/135820762?color=7dcfb6&title=0&byline=0&portrait=0" width="960" height="540" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

Comment:
Can we use the video from our YT channel here? https://www.youtube.com/watch?v=qITgGuvCKeY